### DISCUSSION TITLE
FX EAST USA

### DISCUSSION THREAD

**Discussion by RT5666** [iptv-org/community#807] [Q&A] [answered]
### Channel Name

FX EAST (USA)

### Channel ID

FXEast.us

### Website

http://www.fxnetworks.com/

### Notes

Popular for What We Do in the Shadows series. 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment, marked as answer]
https://github.com/iptv-org/iptv/issues/19784
